src/Mvc/Mvc.Core/src/ModelBinding/CompositeValueProvider.cs

```diff
@@ -80,6 +80,22 @@ public static async Task<CompositeValueProvider> CreateAsync(ControllerContext c
             return new CompositeValueProvider(valueProviderFactoryContext.ValueProviders);
         }
 
+        internal static async ValueTask<(bool success, CompositeValueProvider valueProvider)> TryCreateAsync(
```

Comment:
This is effectively a utilty method which is why this isn't public. It's just convenient to have it sit here.